### ISSUE TITLE
Minor issue: no return statement in function returning non-void in tests code

### ISSUE THREAD

**Issue by Jocelyn-MAYER** [libssh2/libssh2#798]
When compiling libssh2, a warning is emitted:
libssh2-1.10.0/tests/openssh_fixture.c:218:1: warning: no return statement in function returning non-void [-Wreturn-type]

Here is a simple way to avoid this warning, should be correct in the POSIX case, I cannot tell for the WIN32 API:
```
--- libssh2-1.10.0/tests/openssh_fixture.c      2021-08-28 23:15:14.000000000 +0200
+++ libssh2-1.10.0/tests/openssh_fixture.c      2023-02-16 22:29:11.450599958 +0100
@@ -212,8 +212,10 @@ static unsigned int portable_sleep(unsig
 {
 #ifdef WIN32
     Sleep(seconds);
+
+    return 0;
 #else
-    sleep(seconds);
+    return sleep(seconds);
 #endif
 }
```

Please review and apply...

**Comment by dfandrich** [comment]
That issue has already been fixed in commit f0417cb1 by making the function return void.

**Comment by willco007** [comment]
Closing as already fixed in main.
